.husky/commit-msg

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
-if ! head -1 "$1" | grep -qE "^(feat|fix|ci|chore|docs|test|style|refactor|perf|build|revert)(\(.+?\))?: .{1,}$"; then
-    echo "🚫 Aborting commit. Your commit message is invalid. Please, follow the conventional commits spec (https://www.conventionalcommits.org/en/v1.0.0/) " >&2
+if ! head -1 "$1" | grep -qE "^(feat|fix|ci|chore|docs|test|style|refactor|perf|build|revert)(\(.+?\))?: .{1,}$" && \
+   ! head -1 "$1" | grep -qE "^Version Packages"; then
```

Comment:
Should we/can we make it `ci: Version Packages`, to match conventional commits?

Reply:
I'll try it now